pkg/binary/envoytest/util.go

```diff
@@ -85,16 +62,5 @@ func RequireRunTerminate(t *testing.T, terminate func(r *envoy.Runtime), r *envo
 		t.Fatal("Run never completed")
 	case <-ctx.Done():
 	}
-}
-
-// RequireRestoreWorkingDir restores the working directory from the debug archive and returns the archive name.
-func RequireRestoreWorkingDir(t *testing.T, workingDir string, c interface{}) string {
-	// Run deletes the debug store directory after making a tar.gz with the same name.
-	// Restore it so assertions can read the contents later.
-	debugArchive := filepath.Join(workingDir + ".tar.gz")
-	defer os.Remove(debugArchive) //nolint
-
-	e := archiver.Unarchive(debugArchive, filepath.Dir(workingDir)) // Dir strips the RunID directory name
-	require.NoError(t, e, "error restoring %s from %s after stopping [%v]", workingDir, debugArchive, c)
-	return debugArchive
+	return err // nil unless r.Run erred
```

Comment:
nit you can omit the explicit return when you declare return values in function declaration 

```suggestion
	return
```